source/admin_manual/migrating_mailboxes.rst

```diff
@@ -0,0 +1,314 @@
+This depends entirely on the source system. See :ref:`authentication-master_users` for more details.
+
+Preparations
+------------
+
+Configure authentication on the source system to match your preference. Ensure both source and target system agree on usernames.
+
+Configure IMAP client on the target system.
+
+Common settings::
+
+  # these are supported by standard adhering servers
+  imapc_features = rfc822.size fetch-headers
+
+  # If the old IMAP server uses INBOX. namespace prefix, set:
+  #imapc_list_prefix = INBOX
+
+  # Remote hostname
+  imapc_host = remotehost
+
+  ## if you are using TLS
+  imapc_ssl = imaps
+  imapc_port = 993
+
+  # if you are using StartTLS
+  imapc_ssl = starttls
+  imapc_port = 143
+
+  ## these default to system
+  ssl_client_ca_dir = /etc/ssl/certs
+  # or
+  ssl_client_ca_file = /etc/ssl/ca-certificates.pem
+
+Master password::
+
+  imapc_user = %u
+  imapc_password = supersecret
+
+Master user::
+
+  imapc_user = %u
+  imapc_master_user = master-user
+  imapc_password = master-password
+
+Individual password::
+
+  imapc_user = %u
+
+  doveadm -o imapc_password=password sync -1Ru user imapc:
+
+You can verify that the settings are done correctly with::
+
+  doveadm -o imapc_password=password -o mail_location=imapc: mailbox list -u user
+
+POP3 notes
+----------
+
+If you need to retain POP3 support on your new system, you should use :ref:`plugin-pop3-migration`.
+
+::
+
+  pop3c_host = hostname
+
+  # Authenticate as masteruser / masteruser-secret, but use a separate login user.
+  # If you don't have a master user, remove the pop3c_master_user setting.
+  pop3c_user = %u
+  pop3c_master_user = masteruser
+  pop3c_password = masteruser-secret
+
+  # if you are using TLS
+  pop3c_ssl = pop3s
+  pop3c_port = 995
+
+  # if you are using StartTLS
+  pop3c_ssl = starttls
+  pop3c_port = 110
+
+  # see imapc migration for
+
+  namespace {
+    prefix = POP3-MIGRATION-NS/
+    location = pop3c:
+    list = no
+    hidden = yes
+  }
+
+  protocol doveadm {
+    mail_plugins = $mail_plugins pop3_migration
+  }
+
+  plugin {
+    pop3_migration_mailbox = POP3-MIGRATION-NS/INBOX
+  }
+
+You can alternatively configure compatible UIDL format in Dovecot. See :ref:`setting-pop3_uidl_format`.
+
+Executing migration
+-------------------
+
+To migrate users, use::
+
+   doveadm -o imapc_password=bar -o pop3c_password=bar sync -1R -u user imapc:
+
+If you are experiencing problems, enable debugging with the -D parameter::
+
+  doveadm -D -o imapc_password=bar -o pop3c_password=bar sync -1Ru username imapc:
+
+Note that Public and Shared namespaces are synchronized automatically (see caveats).
+
+Caveats
+-------
+
+  * You cannot migrate **to** imap, only from.
+
+POP3 caveats
+------------
+  * POP3 message order (when it's different from IMAP message order) is currently preserved only when destination is Maildir, sdbox or mdbox. It is not preserved with mbox format.
```

Comment:
Better to just say "... is currently not preserved with mbox format". Because otherwise we need to list obox format here, and any other future mailbox format. It's just mbox that will never support this.

Reply:
oh, and leave out the "currently" word as well. We'll never support it.